### ISSUE TITLE
ZStandard Presets

### ISSUE THREAD

**Issue by indolering** [powturbo/TurboBench#22]
Your default levels for ZSTD seem a little wonky:

- TURBO: ZSTD is missing, it should be `-5` | `--fast=5` (the fastest level **currently** available).   This should [push it into LZ4 territory](https://github.com/facebook/zstd/releases/tag/v1.3.4).
- EFFICIENT should probably include 3, ZSTD's default level.
- OPTIMAL (meaning best trade off?) should probably be 20, as 22 balloons memory usage and requires another flag in order to use it.

You might want to comment the .ini file, as it's not clear what each preset means (like BWT).  Also, the [README mentions](https://github.com/powturbo/TurboBench#--groups-fastestfastefficientmaxoptimalbwt) includes a FASTEST preset the [turbobench.ini file](https://github.com/powturbo/TurboBench/blob/e7a73f31b4ad356e8a7fa55e385476627f679c8f/turbobench.ini#L1) only has TURBO.

**Comment by powturbo** [comment]
turbobench.ini is only an example, you can make your own preferred settings.

**Comment by indolering** [comment]
If this is what you use for the headline results listed in the project's README, then leaving the defaults as-is misleads the reader.

**Comment by powturbo** [comment]
I'll test and add your options for zstd when I make a new update the next time

**Comment by indolering** [comment]
Great!  Looking over things again, I think I need to refactor my recommendations.

I'm not sure your intent for the TURBO/FAST/EFFICIENT/MAX/OPTIMAL categories are, but here is what settings I *think* match what you are going after and show off the full range of ZSTD performance points:

- TURBO: The fastest possible?  Apparently, ZSTD supports `--fast=7` now.  I'm away from my Linux machine, but [it looks like](https://github.com/facebook/zstd/blob/7868f38019d3e9e83651b5dc4118c4df25440b5f/tests/playTests.sh#L198) supplying an even higher number would be automatically fixed.  You could supply 10 or 99 if you don't want to bother fixing it everytime ZSTD releases a new `--fast` setting.
- FAST: it seems like FAST would be `--fast=1`, the default when passing just `--fast=1`.  Maybe level 1 as well?
- EFFICIENT: probably 3 (the default) and 5.
- You use the `22` preset in both MAX and OPTIMAL.  `22` is a non-standard setting that [third party decoders are allowed to reject](https://github.com/facebook/zstd/issues/435) because it can result in a DOS attack.  It should still be included as it shows the maximum compression ZSTD can do, but probably not in both settings....  `19` is the universally compatible variant.

TL;DR: `--fast=99/1`, `-1/3/5/19`, and `--ultra -22`
